packages/terra-clinical-item-view/tests/jest/ItemView.test.jsx

```diff
@@ -3,6 +3,13 @@ import React from 'react';
 import { shallowWithIntl } from 'terra-enzyme-intl';
 import ItemView from '../../lib/ItemView';
 
+jest.mock('uuid', () => ({ v4: () => '00000000-0000-0000-0000-000000000000' }));
```

Comment:
I believe recently there was a [PR](https://github.com/cerner/terra-clinical/pull/911/files#diff-5c8f054737c8e43c7ebd6f698efb698903d7bf556d1add6c6ab24425acbb5f47R64-R71) to change jest.mock to jest.spyOn. Can we use the spyOn here as well?